packages/core/src/i18n/locales/en.ts

```diff
@@ -104,6 +104,12 @@ export const en = {
       aliases: ["file", "upload", "embed", "media", "url"],
       group: "Media",
     },
+    emoji: {
+      title: "Emoji",
+      subtext: "Used for inserting an emoji",
```

Comment:
I think we should get rid of the "Used for" part in these sentences. Imo it doesn't add any value.

Compare to notion:

<img width="332" alt="image" src="https://github.com/TypeCellOS/BlockNote/assets/368857/6841250e-8bee-4e02-b2ab-97c853169ca0">